app/scripts/modules/kubernetes/securityGroup/details/details.html

```diff
@@ -75,12 +75,35 @@ <h3 select-on-dbl-click>
         No rules created
       </dl>
     </collapsible-section>
+    <collapsible-section heading="TLS">
+      <dl ng-class="insightCtrl.vm.filtersExpanded ? '' : 'dl-horizontal dl-medium'"
+          ng-repeat="tls in securityGroup.ingress.spec.tls"
+          ng-if="securityGroup.ingress.spec.tls && securityGroup.ingress.spec.tls.length > 0">
+        <dt>Host</dt>
+        <dd>{{tls.hosts[0]}}</dd>
+      </dl>
+      <dl ng-if="!securityGroup.ingress.spec.rules || securityGroup.ingress.spec.rules.length == 0">
```

Comment:
s/==/===

Reply:
Actually, you can just make this a single condition, since you won't hit any NPEs in a template:
`ng-if="!securityGroup.ingress.spec.rules.length"`